server/modules/pool.js

```diff
@@ -21,8 +21,8 @@ if (process.env.DATABASE_URL) {
   };
 } else {
   config = {
-    // user: null || 'con',
-    // password: null || 'secretpass',
+    user: null || 'con',
```

Comment:
i might just be blanking, but what is this for?

Reply:
oh thats just for my use in databses, i have to give stuff username and passwords, do my best to comment it out before i send anything to github so no one has to figure out what it is

Reply:
oh cool, that rings a bell-thanks for clarifying, if it ever makes it through a pull I'll know what to do